modules/oc-mirror-updating-cluster-manifests-v2.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-updating-cluster-manifests-v2_{context}"]
+= Configuring your cluster to use the resources generated by v2
+
+After you have mirrored your image set to the mirror registry, you must apply the generated ImageDigestMirrorSet (IDMS), ImageTagMirrorSet (ITMS), CatalogSource, and release image signature resources into the cluster.
+
+The ImageDigestMirrorSet resource associates the mirror registry with the source registry and redirects image pull requests using digest specifications. Whereas, the ImagetagMirrorSet redirects the image pull requests using image tags. The CatalogSource resource is used by Operator Lifecycle Manager (OLM) to retrieve information about the available Operators in the mirror registry. The release image signatures are used to verify the mirrored release images.
```

Comment:
This is duplicated from the previous section

Reply:
Removed